security/jwt/src/test/java/io/quarkus/ts/security/jwt/BaseJwtSecurityIT.java

```diff
@@ -69,32 +53,32 @@ public void securedEveryoneAdminGroup() throws Exception {
     }
 
     @Test
-    public void securedEveryoneWrongIssuer() throws Exception {
-        givenWithToken(createToken(Invalidity.WRONG_ISSUER))
+    public void securedEveryoneWrongIssuer() {
+        givenWithToken(createToken(Invalidity.WRONG_ISSUER, EMPTY_GROUP))
                 .get("/secured/everyone")
                 .then()
                 .statusCode(HttpStatus.SC_UNAUTHORIZED);
     }
 
     @Test
     public void securedEveryoneWrongDate() throws Exception {
```

Comment:
```suggestion
    public void securedEveryoneWrongDate() {
```
The declared exceptions are redundant in all present methods.